sprint1/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="header-container">
+            <a class="header-logo" href="index.html"></a>
+            <a class="header-logo-mobile" href="index.html"></a>
+            <a class="header-button" href="./login/login.html">로그인</a>
+        </div>
+    </header>
+    <main>
+        <div class="main-top">
+            <div class="main-top-text-container">
+                <div class="main-top-text">일상의 모든 물건을  <br>거래해 보세요</div>

```

Comment:
❗️ 수정요청
지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
                <div class="main-top-text">일상의 모든 물건을<br>거래해 보세요</div>
```